backend/Bindings/Edna.Bindings.User/UserBindingExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Edna.Utils.Http;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Edna.Bindings.User
+{
+    public static class UserBindingExtensions
+    {
+        public static IWebJobsBuilder AddUserBinding(this IWebJobsBuilder builder)
+        {
+            builder.AddExtension<UserExtensionConfigProvider>();
+
+            builder.Services.AddHttpClient();
```

Comment:
This line seems duplicate of the line below it. Do we need to call `AddHttpClient()` twice?

Reply:
this is how it is being done in the other bindings, so I went ahead and did it similarly here also. I'll try removing it and see if has any impact or not

Reply:
All the best with your explorations 🥇. Hope we learn something new with this excercise.

Reply:
I believe these explorations wrt to the Bindings can be kept as good first issues for now and be utilized to help onboard someone new in the project. What do you suggest?

Reply:
Whatever feels best to you. Unfortunately, I don't prefer having investigations as github first issues unless they have some customer facing / testable impact. But yes, I agree that we can keep this as a separate PR.

Reply:
I'd say it would be best to choose from your personal preference. I'd refrain from putting explorations as issues unless there's a customer impact. Saying that, I'm totally fine with keeping this out of this pr, and maybe take a look at this refactoring later on.